fetch_depth_layer/src/depth_layer.cpp

```diff
@@ -232,10 +232,10 @@ void FetchDepthLayer::depthImageCallback(
     // Get normals
     if (normals_estimator_.empty())
     {
-      normals_estimator_ = new RgbdNormals(cv_ptr->image.rows,
-                                           cv_ptr->image.cols,
-                                           cv_ptr->image.depth(),
-                                           K_);
+      normals_estimator_.reset(new RgbdNormals(cv_ptr->image.rows,
+                                               cv_ptr->image.cols,  
```

Comment:
```suggestion
                                               cv_ptr->image.cols,
```

Reply:
remove extra whitespace and then this can be merged via the GitHub `squash & merge` feature to cleanup the commit message from the first commit